addons/mail/static/tests/chat_bubble/chat_bubble.test.js

```diff
@@ -338,13 +338,15 @@ test("More than 7 actually folded chat windows shows a 'hidden' chat bubble menu
 
 test("Can close all chat windows at once", async () => {
     const pyEnv = await startServer();
+    const channelIds = [];
     for (let i = 1; i <= 20; i++) {
-        pyEnv["discuss.channel"].create({
+        const channelId = pyEnv["discuss.channel"].create({
             name: String(i),
             channel_member_ids: [
                 Command.create({ fold_state: "folded", partner_id: serverState.partnerId }),
             ],
         });
+        channelIds.push(channelId);
     }
```

Comment:
This should be a single create call, the param should be an array built with map.